azerothcore/azerothcore-wotlk
# fix(DB/Creature): remove mobs from brewfest camp

## CHANGES PROPOSED:
-  remove hostile and non-hostile mobs from brewfest camp


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3496



## TESTS PERFORMED:
- Applied sql and restart server
- Screenshot of test in https://github.com/azerothcore/azerothcore-wotlk/issues/3496



## HOW TO TEST THE CHANGES:

1. Go to the brewfest camp right outside orgrimmar
2. Check if you seen any hostile or non-hostile mobs within the camp area 

## KNOWN ISSUES AND TODO LIST:




## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
